store/src/java/com/zimbra/cs/zclient/ZMailboxUtil.java

```diff
@@ -2677,14 +2678,26 @@ private void doGetContacts(String[] args) throws ServiceException {
     private void doGetContactFrequency(String[] args) throws ServiceException {
         String email = args[0];
         String freqBy = args[1];
-        GetContactFrequencyResponse resp = mMbox.getContactFrequency(email, freqBy);
+        GetContactFrequencyResponse resp;
+        if(args.length > 2) {
+            Integer offsetInMinutes = Integer.parseInt(args[2]);
+            resp = mMbox.getContactFrequency(email, freqBy, offsetInMinutes);
+        } else {
+            resp = mMbox.getContactFrequency(email, freqBy);
+        }
         for (ContactFrequencyData data: resp.getFrequencyGraphs()) {
             dumpFrequencyGraph(data);
         }
     }
 
     private void dumpFrequencyGraph(ContactFrequencyData graphData) {
         String freqBy = graphData.getFrequencyBy();
+        for (ContactFrequencyDataPoint dataPoint : graphData.getDataPoints()) {
+            String unixTimestamp = dataPoint.getLabel();
+            Instant utcTimestamp = Instant.ofEpochMilli(Long.parseLong(unixTimestamp));
+            dataPoint.setLabel(utcTimestamp.toString());
+        }
+        stdout.println("Epoch milliseconds are included in the actual response. UTC timestamp shown here are just for demo");
```

Comment:
I'd shorten this to "date labels shown in UTC; other clients may show the browser's timezone instead"